rt-preempt/ubuntu18.04/install_rt_preempt_kernel.sh

```diff
@@ -61,6 +62,9 @@ read -p "Please read the above instructions" yn
 
 make menuconfig -j
 
+# Disable the SYSTEM_TRUSTED_KEYS from the config.
+# SEE: https://askubuntu.com/a/1329625
+scripts/config --disable SYSTEM_TRUSTED_KEYS
```

Comment:
how does this work?
Is there a scripts/config executable available?

Reply:
yes, there is a `scripts/config` at the current location when executing the script / in the linux kernel sources.